src/common/utils/enums.ts

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+export type StrEnumValue<T> = T[keyof T];
+export type StrEnum<T> = Record<keyof T, string>;
+
+function getEnumStrValues<T extends StrEnum<T>>(
+  enumType: T,
+): Array<StrEnumValue<T>> {
+  return Object.values(enumType)
+    .filter((v) => typeof v === 'string')
+    .map((v) => v as StrEnumValue<T>);
+}
+
+export function isInStrEnum<T extends StrEnum<T>>(
+  value: unknown,
+  enumType: T,
+): boolean {
+  for (const enumValue of getEnumStrValues(enumType)) {
+    if (enumValue === value) {
+      return true;
+    }
+  }
+  return false;
+}
```

Comment:
Minor, but this could be a filter?

```js
function isInStrEnum(value, enumType) {
  return getEnumStrValues(enumType).filter(
    (x) => x === value)
  )
}
```

Reply:
If you want it be a way you like, just turn it into a github suggestion and I'll click the button.

Reply:
Fixed

Reply:
I fixed the code, the suggestion used `filter()` but that needs to be `every()`

Reply:
Changed this to a find() call with boolean coercion from filter() since filter() returns an arrray of found values.